discovery-infra/test_infra/consts/consts.py

```diff
@@ -92,13 +92,6 @@ class NetworkType:
 DEFAULT_TEST_INFRA_DOMAIN = f".{CLUSTER_PREFIX}-{DEFAULT_NAMESPACE}.{DEFAULT_BASE_DNS_DOMAIN}"
 TEST_TARGET_INTERFACE = "vnet3"
 SUFFIX_LENGTH = 8
-RELEASE_IMAGES_PATH = "assisted-service/data/default_release_images.json"
-
-DEFAULT_IPV6_SERVICE_CIDR = "2003:db8::/112"
-DEFAULT_IPV6_CLUSTER_CIDR = "2002:db8::/53"
-DEFAULT_IPV6_HOST_PREFIX = 64
-DEFAULT_PROXY_SERVER_PORT = 3129
-DEFAULT_LOAD_BALANCER_PORT = 6443
```

Comment:
How is it possible it's not used?
ipv6 tests use this

Reply:
So these were removed as part of https://github.com/openshift/assisted-test-infra/pull/1136
Just added back by mistake on rebase.